background-remover/background-remover.py

```diff
@@ -0,0 +1,20 @@
+import os
+import rembg
+
+def process (input_path: str, output_path: str, session = rembg.new_session()):
+    if not os.path.exists(input_path):
+        print(f"ERROR: input file does not exist [{input_path}]")
+        return False
+    
+    output_dir_path = os.path.dirname(output_path)
+    if not os.path.isdir(output_dir_path):
+        print(f"ERROR: output directory does not exist [{output_dir_path}] ")
+        return False
+
+    with open(input_path, 'rb') as i:
+        with open(output_path, 'wb') as o:
```

Comment:
you should not lock the input and output file handles while processing the content - because it potentially might take a long time, instead:

1. open input file and read content
2. close input file handle
3. process input
4. open output file handle
5. write the result
6. close output file handle

Reply:
good idea